foxy/ci-benchmark.yaml

```diff
@@ -0,0 +1,111 @@
+%YAML 1.1
+# ROS buildfarm ci-build file
+---
+build_environment_variables:
+  CMAKE_PREFIX_PATH: '/opt/ros/noetic:$CMAKE_PREFIX_PATH'
+  LD_LIBRARY_PATH: '/opt/ros/noetic/lib:$LD_LIBRARY_PATH'
+  NDDSHOME: '/opt/rti.com/rti_connext_dds-5.3.1'
+  PYTHONPATH: '/opt/ros/noetic/lib/python3/dist-packages:$PYTHONPATH'
+  ROS_PYTHON_VERSION: '3'
+  RTI_NC_LICENSE_ACCEPTED: 'yes'
+benchmark_patterns:
+- ws/test_results/**/*.benchmark.json
+benchmark_schema: !include ../common/benchmark_schema.yaml
+build_tool: colcon
+build_tool_args: '--cmake-args -DAMENT_RUN_PERFORMANCE_TESTS=ON -DCMAKE_BUILD_TYPE=Release --no-warn-unused-cli'
+build_tool_test_args: '--ctest-args -L performance --pytest-args -m performance'
+install_packages:
+- default-jdk # for CycloneDDS
+- libasio-dev # for FastRTPS
+- libtinyxml2-dev # for FastRTPS
+- maven # for CycloneDDS
+- ros-noetic-common-msgs
+- ros-noetic-rosbash
+- ros-noetic-roscpp
+- ros-noetic-roscpp-tutorials
+- ros-noetic-roslaunch
+- ros-noetic-rosmsg
+- ros-noetic-rospy-tutorials
+- ros-noetic-tf2-msgs
+jenkins_job_label: ci-agent
+jenkins_job_priority: 50
```

Comment:
This is the wrong job priority for Foxy. I believe 57 is the right priority.

Reply:
Please post a list of all priorities to ease the review.

Reply:
Updated: https://github.com/ros2/ros_buildfarm_config/pull/130/commits/955765e795a5c7db778c0e67f868315f0b251af8

By the way, Where I can check this information ?

Reply:
> Where I can check this information ?

With the `list_priorities.py` script in the root of this repository.

Reply:
yes, `foxy` jobs are 57:

```bash
57
  jenkins_commit_job_priority::foxy/source-build.yaml
  jenkins_job_priority::foxy/ci-nightly-connext.yaml
  jenkins_job_priority::foxy/ci-nightly-cross-vendor-connext-cyclonedds.yaml
  jenkins_job_priority::foxy/ci-nightly-cross-vendor-connext-fastrtps-dynamic.yaml
  jenkins_job_priority::foxy/ci-nightly-cross-vendor-connext-fastrtps.yaml
  jenkins_job_priority::foxy/ci-nightly-cross-vendor-cyclonedds-fastrtps-dynamic.yaml
  jenkins_job_priority::foxy/ci-nightly-cross-vendor-cyclonedds-fastrtps.yaml
  jenkins_job_priority::foxy/ci-nightly-cross-vendor-fastrtps-fastrtps-dynamic.yaml
  jenkins_job_priority::foxy/ci-nightly-cyclonedds.yaml
  jenkins_job_priority::foxy/ci-nightly-debug.yaml
  jenkins_job_priority::foxy/ci-nightly-extra-rmw-release.yaml
  jenkins_job_priority::foxy/ci-nightly-fastrtps-dynamic.yaml
  jenkins_job_priority::foxy/ci-nightly-fastrtps.yaml
  jenkins_job_priority::foxy/ci-nightly-performance.yaml
  jenkins_job_priority::foxy/ci-nightly-release.yaml
  jenkins_job_priority::foxy/ci-overlay.yaml
```